load-issues.js

```diff
@@ -0,0 +1,21 @@
+var Issue = require('./issue.js');
+var config = require('./config.js');
+var debug = require('debug')('backlog:load-issues');
+var github = require('./github.js');
+
+module.exports = function getIssues() {
+   return github.getOpenBacklogIssues()
+   .then(createIssuesFromGithubResponses)
+}
+
+function createIssuesFromGithubResponses(githubIssues) {
+   return githubIssues.map(createIssueFromGithubResponse);
+}
+
+function createIssueFromGithubResponse(ghIssue) {
+   var issue = new Issue(ghIssue.number);
+   ghIssue.labels.forEach((label) => {
+      issue.addLabel(label.name, 1);
```

Comment:
The literal `1` here is totally opaque to me.

Reply:
Good point. Fixed with a comment. I went down the road of removing the `appliedOn` argument to `addLabel()` since it's not used in sorting. It was a lot of little changes and the code wasn't much simpler. Filling in the correct data from the issue events wouldn't be that hard though.